docs/clay_over_aoi.ipynb

```diff
@@ -0,0 +1,727 @@
+       "Datum: World Geodetic System 1984 ensemble\n",
+       "- Ellipsoid: WGS 84\n",
+       "- Prime Meridian: Greenwich"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mgrs.crs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5211112-d169-40b4-83af-1a7cc40c2439",
+   "metadata": {},
+   "source": [
+    "## Create a Geopandas dataframe with AOI\n",
+    "\n",
+    "This example uses a string with a single polygon over the area around Puri, India."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2cb85d85-1eec-4927-8c53-71e68033a942",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "image/svg+xml": [
+       "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"100.0\" height=\"100.0\" viewBox=\"84.7647758727867 19.389826947560604 1.3910244317380602 1.2259414120750947\" preserveAspectRatio=\"xMinYMin meet\"><g transform=\"matrix(1,0,0,-1,0,40.0055953071963)\"><path fill-rule=\"evenodd\" fill=\"#66cc99\" stroke=\"#555555\" stroke-width=\"0.027820488634761206\" opacity=\"0.6\" d=\"M 85.05032876899293,19.49499830249873 L 85.16974995888492,19.44134637095831 L 85.96802851682074,19.799945705366934 L 86.02974282300654,20.15322442052609 L 86.10428088112705,20.516965603502392 L 85.40458491618931,20.56424893623799 L 84.94533430002747,20.339711117597215 L 84.8162952961844,19.799945705366934 L 85.05032876899293,19.49499830249873 z\" /></g></svg>"
+      ],
+      "text/plain": [
+       "<POLYGON ((85.05 19.495, 85.17 19.441, 85.968 19.8, 86.03 20.153, 86.104 20....>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "aoi_src = \"\"\"{\n",
+    "    \"type\": \"FeatureCollection\",\n",
+    "    \"name\": \"puri\",\n",
+    "    \"crs\": {\n",
+    "        \"type\": \"name\",\n",
+    "        \"properties\": { \"name\": \"urn:ogc:def:crs:OGC:1.3:CRS84\" }\n",
+    "    },\n",
+    "    \"features\": [\n",
+    "        {\n",
+    "            \"type\": \"Feature\",\n",
+    "            \"properties\": {\n",
+    "                \"Region\": \"Puri\"\n",
+    "            },\n",
+    "            \"geometry\": {\n",
+    "                \"type\": \"Polygon\",\n",
+    "                \"coordinates\": [ [ [ 85.050328768992927, 19.494998302498729 ], [ 85.169749958884921, 19.441346370958311 ], [ 85.968028516820738, 19.799945705366934 ], [ 86.029742823006544, 20.15322442052609 ], [ 86.104280881127053, 20.516965603502392 ], [ 85.404584916189307, 20.564248936237991 ], [ 84.945334300027469, 20.339711117597215 ], [ 84.816295296184407, 19.799945705366934 ], [ 85.050328768992927, 19.494998302498729 ] ] ]\n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "}\"\"\"\n",
+    "aoi = gpd.read_file(aoi_src, driver=\"GeoJSON\")\n",
+    "aoi.geometry[0]"
```

Comment:
Can we show how to specify the AOI via [`shapely.Point`](https://shapely.readthedocs.io/en/2.0.2/reference/shapely.Point.html) or [`shapely.box`](https://shapely.readthedocs.io/en/2.0.2/reference/shapely.box.html) instead? GeoJSON is a little harder to construct manually.

Reply:
Good point, changed to use `shapely.box`